package.json

```diff
@@ -503,7 +503,7 @@
     "test": "run-s test:*",
     "test:lint": "eslint src tests --ext ts",
     "test:lint:fix": "eslint src tests --ext ts --fix",
-    "test:unit": "vitest -c ./vitest.conf.ts",
+    "test:unit": "vitest -c ./vitest.conf.ts --run",
```

Comment:
Maybe separate PR for this? I kind of liked watch mode tests by default

Reply:
reverted